soccer/planning/SingleRobotPathPlanner.hpp

```diff
@@ -20,7 +21,9 @@ class SingleRobotPathPlanner {
     virtual std::unique_ptr<Path> run(
         MotionInstant startInstant, const MotionCommand* cmd,
         const MotionConstraints& motionConstraints,
-        const Geometry2d::ShapeSet* obstacles,
+        Geometry2d::ShapeSet& obstacles,
```

Comment:
Well, I changed it to not const cause I add to the static obstacles. We create unique obstacle sets for each robot anyways.